src/msix/pack/AppxBundleWriter.cpp

```diff
@@ -0,0 +1,240 @@
+//
+//  Copyright (C) 2019 Microsoft.  All rights reserved.
+//  See LICENSE file in the project root for full license information.
+// 
+
+#include "AppxPackaging.hpp"
+#include "AppxBundleWriter.hpp"
+#include "MsixErrors.hpp"
+#include "Exceptions.hpp"
+#include "ContentType.hpp"
+#include "Encoding.hpp"
+#include "ZipObjectWriter.hpp"
+#include "AppxManifestObject.hpp"
+#include "ScopeExit.hpp"
+#include "FileNameValidation.hpp"
+#include "StringHelper.hpp"
+
+#include <string>
+#include <memory>
+#include <future>
+#include <algorithm>
+#include <functional>
+
+#include <zlib.h>

```

Comment:
Do you need this? I would expect that the zipwriter encapsulates all of the zlib code

Reply:
Build fails without zlib.h. Removed the other two.

Reply:
Can you move the #include <zlib.h> to ZipWriter.hpp? 

---
In reply to: [516882961](https://github.com/microsoft/msix-packaging/pull/389#discussion_r516882961) [](ancestors = 516882961)

Reply:
Moved #include <zlib.h> to ZipWriter.cpp.